tools/bumbershoot/idpqonvert/idpqonvert.xml

```diff
@@ -0,0 +1,228 @@
+<?xml version="1.0"?>
+<tool id="idpqonvert" name="idpQonvert" version="0.1.0">
+    <description>Prepare identification results for IDPicker</description>
+    <macros>
+        <xml name="normalizationOptions">
+            <option value="off" selected="true">None</option>
+            <option value="quantile">Quantile</option>
+            <option value="linear">Linear</option>
+        </xml>
+    </macros>
+    <requirements>
+        <requirement type="package" version="3.0.8789">bumbershoot</requirement>
+    </requirements>
+    <stdio>
+        <exit_code range="1:" level="fatal" description="Job Failed" />
+        <regex match="^Error:.*$" source="both" level="fatal" />
+    </stdio>
+    <command>
+<![CDATA[
+        #set $db_name = $input_database.display_name.replace(".fasta", "") + ".fasta"
+        #set $input_name = $input.display_name
+        #set $output_name = $input_name.split(".")[0] + ".idpDB"
+        ln -s '$input' '${input_name}';
+        ln -s '$input_database' '${db_name}';
```

Comment:
Makes sense. I was just following what was there in galaxyp, but I'll change it.